client/src/components/DisplayTable/DisplayTable.js

```diff
@@ -108,9 +108,18 @@ export class DisplayTable extends React.Component {
       })
     );
     const NoDataMessage = () => (
-      <div className="well large_panel_text no-data-msg">
-        {text_maker("no_data")}
-      </div>
+      <tbody>
+        <tr>
+          <td style={{ padding: 0 }} colSpan={_.size(data[0])}>
```

Comment:
This doesn't actually matter but `_.size(data[0])` should be equivalent as `visible_col_keys.length`. Same thing but doesn't need to do data[0]. Upto you if you wanna make the change or not haha